packages/plugins/users-permissions/server/strategies/users-permissions.js

```diff
@@ -75,14 +75,16 @@ const verify = async (auth, config) => {
       },
     });
 
-    const allowedActions = map('action', publicPermissions);
+    const allowedActions = map('action', publicPermissions).map(toLower);
```

Comment:
Actually I think we maybe keep the case everywhere 🤔

Reply:
It's my preference too